src/components/ArticleByTopic.jsx

```diff
@@ -11,9 +11,8 @@ const [articlesByTopic, setArticlesByTopic] = useState([])
 const {topic} = useParams()
 
 	useEffect(() => {
-		getArtibleByTopic(topic).then((data) => {
+		getArticleByTopic(topic).then((data) => {
```

Comment:
I would recommend reusing a component that renders all articles, similar to your topic component. They are after all doing the exact same thing.